### DISCUSSION TITLE
Why my Pair Extra ordinate badge didn't came up?

### DISCUSSION THREAD

**Discussion by Vijaykv5** [community/community#103496] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I worked on a Issue and made a PR here - 
https://github.com/openmrs/openmrs-esm-patient-chart/pull/1626
It include co-authoring from one of the maintainers as you can see below: 
<img width="1041" alt="Screenshot 2024-02-04 at 11 42 59 AM" src="https://github.com/community/community/assets/94985341/f5b32603-c67a-4dd3-b441-ad95a13d4d36">
Is this what `co-authoring` is? Then why does I didn't receive a badge either?

**Comment by DJLink** [comment]
Someone else has previously offered a guide on how to achieve that one, can you check if it fits all the steps?


> Make a PR with people as your coauthors.
> 
> Tell someone to make a PR with you as a coauthor. Here is how you can add a coauthor, https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors

https://github.com/Schweinepriester/github-profile-achievements/discussions/127#discussioncomment-3148824

**Reply by Vijaykv5** [reply]
Thank you @DJLink :)

**Comment by ghost** [comment, marked as answer]
co authoring is when you make commit with two or more users as authors/co-authors. This can be done by 

```bash
git commit -m "Your commit message" --author="Author Name <author@example.com>" --author="Co-Author Name <coauthor@example.com>"

```

In professional terms,

In collaborative software development, co-authoring refers to multiple individuals contributing to a single Git commit, acknowledging their joint effort on a particular change. This is often used to recognize and attribute contributions from team members who collaborated on a specific feature, bug fix, or improvement.

**Comment by queenofcorgis** [comment]
Hi @Vijaykv5,  

This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
